app/jobs/submit_folio_request_job.rb

```diff
@@ -89,7 +89,7 @@ def create_item_circulation_request(barcode)
 
       request_data = FolioClient::CirculationRequestData.new(
         request_level: 'Item',
-        request_type: item.best_request_type,
+        request_type: item.best_request_type(patron),
```

Comment:
This might end up a little gnarly, but I wonder if this whole refactor would be clearer if we created `Patron#best_request_type(item)`. That should let us calculate the circ rules for the patron, instead of having to do it on the item

As a result, `Folio::CirculationRules::PolicyService.rules` should just be all the rules, and the group filtering can just stay with the instance? I think we were trying to keep the applicable rules slim for Searchworks because it could evaluate some thousands of items on a search results page... but requests might be better off using the whole list.